src/chain/manager.js

```diff
@@ -0,0 +1,68 @@
+import utils from 'ethereumjs-util'
+import {Buffer} from 'safe-buffer'
+
+import chain from './index'
+import Transaction from './transaction'
+import config from '../config'
+
+const rlp = utils.rlp
+
+export async function getAllUTXOs(address) {
+  if (!address || !utils.isValidAddress(address)) {
+    return []
+  }
+
+  const from = Buffer.concat([
```

Comment:
What about a method that would handle the buffer allocation for the block number, tx index, and output index, rather than repeating the 3 lines over and over?

Reply:
Good catch. I'll add a method. Thanks.